DMTR-53.tex

```diff
@@ -147,47 +147,297 @@ \subsection{Summary Table}
+		\texttt{base\_SdssCentroid\_x},
+		\texttt{base\_SdssCentroid\_y},
+		\texttt{base\_SdssCentroid\_xSigma}, \&
+		\texttt{base\_SdssCentroid\_ySigma} \\
+		\hline
+		sky position and associated error (radec) & FAIL & \texttt{coord\_ra} and \texttt{coord\_dec} are present, but no error is reported \\
+		\hline
+	SNR of the detection & FAIL & no SNR is reported \\
+		\hline
+		calibrated PS flux and associated error & PASS & \texttt{base\_PsfFlux\_flux} and \texttt{base\_PsfFlux\_fluxSigma} \\
+		\hline
+	likelihood of the observed data given the PS model & False & not present in the catalog \\
+		\hline
+		calibrated aperture flux and associated error & PASS & several values of \texttt{base\_CircularApertureFlux\_*} are present with errors \\
+		\hline
+	calibrated flux and associated error for a trailed source model & FAIL & not present in the catalog \\
+		\hline
+	length and angle of the trail & FAIL & not present in the catalog \\
+		\hline
+		flux and associated parameters for a dipole model & PASS & several dipole flux values are present \\
+		\hline
+		parameters of an adaptive shape measurement and associated error & PASS & \texttt{base\_SdssShape\_*} \\
+		\hline
+	a measure of source extendedness & FAIL & not present in the catalog \\
+		\hline
+	the estimated background at the position of the object in the template image with associated uncertainty & FAIL & not present in the catalog \\
+		\hline
+	a measure of spuriousness & FAIL &  not present in the catalog \\
+		\hline
+	flags indicating problems encountered while computing the aforementioned attributes & PASS & many flags are present \\
+		\hline
+  calibrated flux and associated error for the DIAsource as measured on the Visit image & FAIL & not present in the catalog \\
+		\hline
+\end{tabular}
+	\caption{Required DIASource attributes for AG-00-20 \label{tab:diasrc}}
+\end{table}
+
+
+\textit{Faint DIASources satisfiying additional criteria are stored: FAILED}
+
+No specification for the ``additional criteria'' necessary to store 
+sub-threshold DIASources are available at this time; 
+this is a failure of requirements rather than of the pipelines.
+
+A new JIRA ticket, DM-12967, has been created to capture this issue.
+
+\textit{Relevant derived quantities are provided in pre-computed columns: PASSED}
+
+Examination of the DIASource record shows two entries,
+\texttt{ip\_diffim\_PsfDipoleFlux\_chi2dof} and
+\texttt{ip\_diffim\_DipoleFit\_chi2dof}, which provide goodness of fit measures
+as required.  Color-color diagrams are not relevant for single-visit DIASource
+catalogs. The requirement does not specify an exhaustive list of relevant
+derived quantities and hence cannot be fully tested as written.
+
 \subsection{AG-00-25}
 
-\appendix
+\textit{Measurements are Presented in Flux Units: FAILED}
+
+No measurements (or aggregated measurements, such as means) 
+were contained in the the DIAObject table as implemented at the 
+time of the test.
```

Comment:
We might claim that as a pass: absolutely every relevant measurement was presented in flux units. :-)

Actually, I'd be inclined to record it as a "deviation from the test procedure" that this wasn't tested since no appropriate measurements were made.